administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -188,10 +188,13 @@ COM_LOCALISE_CANNOT_REMOVE_DEFAULT_LANGUAGE="Localise cannot remove default lang
 
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT="Translate"
 COM_LOCALISE_ACCESS_TRANSLATION_EDIT_DESC="Translate description"
+COM_LOCALISE_CHECKBOX_TRANSLATION_GRAMMAR_CASE="Grammar case"
 COM_LOCALISE_CONFIRM_TRANSLATION_SAVE="Before saving, make sure you have set if the translation is 'Complete' or not in the 'Details' tab."
+COM_LOCALISE_CONFIRM_TRANSLATION_SAVE_NOTINREF="Before saving, make sure to review if within the 'Not in ref' section you have selected the corresponding checkboxes of the strings to be deleted."
```

Comment:
Suggest
`COM_LOCALISE_CONFIRM_TRANSLATION_SAVE_NOTINREF="Also, if you have strings in the 'Not in reference' section,  make sure you have selected the corresponding checkboxes of the strings you want to delete."`

Reply:
Done